girder/api/v1/assetstore.py

```diff
@@ -260,3 +261,18 @@ def updateAssetstore(self, assetstore, params):
     def deleteAssetstore(self, assetstore, params):
         self.model('assetstore').remove(assetstore)
         return {'message': 'Deleted assetstore %s.' % assetstore['name']}
+
+    @access.admin
+    @loadmodel(model='assetstore')
+    @describeRoute(
+        Description('Get a list of files controlled by an assetstore.')
+        .param('id', 'The assetstore ID.', paramType='path')
+        .pagingParams(defaultSort='_id')
+        .errorResponse()
+        .errorResponse('You are not an administrator.', 403)
+    )
+    def getAssetstoreFiles(self, assetstore, params):
+        limit, offset, sort = self.getPagingParameters(params, 'lowerName')
```

Comment:
why is there a different sort order within the method `'lowerName'` compared to the `@describeRoute`, `'_id'`?